data/db-schema/src/main/resources/net/corda/db/schema/crypto/migration/crypto-creation-v1.0.xml

```diff
@@ -4,10 +4,20 @@
                    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
     <changeSet author="R3.Corda" id="crypto-creation-v1.0">
         <createTable tableName="crypto_wrapping_key">
+            <column name="id" type="UUID">
+                <constraints nullable="false"/>>
+            </column>
             <column name="alias" type="VARCHAR(64)">
                 <constraints nullable="false"/>
             </column>
-            <column name="created" type="TIMESTAMP">
+            <!-- generation of the wrapping key; higher generation numbers to be used to replaced expiring wrapping keys -->
+            <column name="generation" type="INT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="created" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP">
+                <constraints nullable="false"/>
+            </column>
+            <column name="accessed" type="TIMESTAMP" defaultValueComputed="CURRENT_TIMESTAMP">
```

Comment:
What's the accessed timestamp?
I don't think we should update a column on each read if that is what the intention was.

Reply:
removed